xds/internal/balancer/clusterresolver/resource_resolver_dns.go

```diff
@@ -51,7 +51,7 @@ func newDNSResolver(target string, topLevelResolver *resourceResolver) *dnsDisco
 		target:           target,
 		topLevelResolver: topLevelResolver,
 	}
-	r, err := newDNS(resolver.Target{Scheme: "dns", URL: *testutils.MustParseURL("dns:///" + target)}, ret, resolver.BuildOptions{})
+	r, err := newDNS(resolver.Target{Scheme: "dns", URL: *utils.MustParseURL("dns:///" + target)}, ret, resolver.BuildOptions{})
```

Comment:
The other comment about not calling `panic` from non-test code applies to here as well.